community/server/pom.xml

```diff
@@ -159,7 +159,7 @@
 
     <dependency>
       <groupId>ch.qos.logback</groupId>
-      <artifactId>logback-classic</artifactId>
+      <artifactId>logback-core</artifactId>
```

Comment:
@tinwelint / @benbc We actually do have our own adapter thing for netty, called `Netty4LoggerFactory`, and [it is set here](https://github.com/neo4j/neo4j/blob/3.0/community/server/src/main/java/org/neo4j/server/Bootstrapper.java#L84). Netty _only uses logback for a single message on initialization_, due to an [odd behaviour in the static intializer of the netty `InternalLoggerFactory`](https://github.com/netty/netty/blob/master/common/src/main/java/io/netty/util/internal/logging/InternalLoggerFactory.java#L37-L66) that was added in 4.0. I consider that a bug in netty, and will probably raise it to the netty team.

Given it's only the one use of logback in the static initializer, we could just do the very simple thing in 2.3 of silencing that by doing something like this:

```
        final PrintStream originalStdout = System.out;
        System.setOut( NullPrintStream.getInstance() );
        InternalLoggerFactory.setDefaultFactory( new Netty4LoggerFactory( userLogProvider ) );
        System.setOut(originalStdout);
```

If you'd like to try that approach, I have a commit that does exactly that here: https://github.com/cleishm/neo4j/commit/95f99eaf992b9bdcbd547af773f7b8ba606f544f